doc/manuals/orion-api.md

```diff
@@ -1021,9 +975,13 @@ to keep your client decoupled from implementation details.
         + registrations_url: /v2/registrations (required, string) - URL which points to the
           registrations resource
 
-# Group Entities
+# API Routes
+
+## Entities Operations
+
+### Entities List
 
-### List Entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,typePattern,q,mq,georel,geometry,coords,attrs,metadata,orderBy}]
+#### List Entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,typePattern,q,mq,georel,geometry,coords,attrs,metadata,orderBy}]
```

Comment:
```suggestion
#### List Entities (`GET /v2/entities`)
```

Reply:
done in https://github.com/telefonicaid/fiware-orion/pull/4142/commits/dc7d5dcefa4f8f8a432adae2265cc6c6cb1f663c and https://github.com/telefonicaid/fiware-orion/pull/4142/commits/462fb895e30498c2d42fe25906bb0efbaf0bd701